### PR TITLE
Fix code quality

### PR DESCRIPTION
**Pull request by ajgoranson**

### CODE REVIEW

**Comment by claraj** [comment]
@4LineTime can you review?

**Comment by 4LineTime** [comment]
Looking good! I like the way you changed the menu function!

**Comment by 4LineTime** [comment]
Beautiful!

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,56 @@
+"""
+This python program converts states in abbreviations and abbreviations into states 
+"""
+import json  # import json for processing
 
-
-
-import json  # import json for json processing
 def main():
+    """ Main method """
```
Comment (claraj) [inline convert_state.py:7]: This is a function.   Describe what it does

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
+    print(json_data)
+ 
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    state_name_keys = json_data # dictionary of state abbreviations keys and state name values
```
Comment (claraj) [inline convert_state.py:14]: Use a more specific variable name like states_abbreviations_to_names and then it's more obvious which dictionary is which, and you don't need the comment

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,56 @@
...
+ 
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    state_name_keys = json_data # dictionary of state abbreviations keys and state name values 
+    state_dictionary = {} # dictionary of state name keys and state abbreviation values
```
Comment (claraj) [inline convert_state.py:15]: Use a more specific variable name like state_names_to_abbreviations and then it's more obvious which dictionary is which, and you don't need the comment

File: convert_state.py [hunk 4]
```diff
@@ -1,60 +1,56 @@
...
-        else:
-            print('try again')
-def convertStateToAbbreviation(dictionary ):
-    userInput   = input("Enter state name").  capitalize()
-    result = dictionary.get(userInput  )
-    if result == None:
-        print(' state not found')
+def convert_state_to_abbreviation(state_dictionary):
+    """
+    Takes states and converts to abbreviation
```
Comment (claraj) [inline convert_state.py:35]: More specifically, it asks the user for a state

File: convert_state.py [hunk 5]
```diff
@@ -1,60 +1,56 @@
...
-        print ( "The abbreviation for " +  userInput+" is " + result)
+        print(f'The abbreviation for {user_input} is {result}')
+
 def convert_abbreviation_to_state(dictionary):
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
+    """
+    Take state abbreviation to a state
```
Comment (claraj) [inline convert_state.py:46]: Be more specific

**Review by claraj — CHANGES_REQUESTED** [review]
Looking much better! A few things to look at and fix please.

**Pull request closed without merging**